astro/src/content/docs/apis/_user-email-verification-ids-response.mdx

```diff
@@ -0,0 +1,20 @@
+import APIField from 'src/components/api/APIField.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+<APIField name="emailVerificationId" type="String" since="1.27.0">
+  An email verification Id. When present, this will represent the user's current email verification Id.
+
+  When using <code>FormField</code> verification strategy, this is the first part of the pair of verification Ids, and the <InlineField>emailVerificationOneTimeCode</InlineField> is the second part. When <code>ClickableLink</code> is the verification strategy, this value is sensitive in that this value by itself can be used to verify the user's email address.
```

Comment:
Since these are broken out into fragments we could go hardcore on markdown syntax over html for things like `<code>` and change them all to backticks

Should we do that here?